src/routes/auth/get-invited-user-detail.e2e.test.js

```diff
@@ -0,0 +1,86 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const { fixtures } = require('../../../test/utils')
+
+describe('GET /auth/invites/token', () => {
+
+	describe('Failure', () => {
+
+		let invalidInviteToken
+		let user
+		let accessToken
+
+		before(async () => {
+		
+			invalidInviteToken = fixtures.generateAccessToken({
+				email : faker.internet.email()
+			})
+			user = await fixtures.insertUser() 
+			const body = { id: user.id, email: user.email }
+			accessToken = fixtures.generateAccessToken(
+				{user : body}
+			)
```

Comment:
We only need the invite token for the route to work. It's not an authenticated route so we won't be using this (does not use the `isAuthenticated` route). That is, the user hasn't been created yet at this point so there is no possibility of getting an access token in the `Authorization` header.